src/integrations/feed/configs.ts

```diff
@@ -83,11 +93,11 @@ const addFiltersToConfig = ({
       baseConfig.allowed_post_types || postTypes
     ).filter((x) => !filters.excludeTypes.includes(x));
   }
-  if (
-    filters.blockedContentCuration?.length &&
-    opts.includeBlockedContentCuration
-  ) {
+  if (filters.blockedContentCuration?.length && opts.includeContentCuration) {
     baseConfig.blocked_content_curations = filters.blockedContentCuration;
```

Comment:
I think we don't need this anymore now

Reply:
What about the lofn provider? (And maybe the one that uses the digest?)

@vpol you probably know best, can we remove it?

Reply:
lofn does not use any of the filters at all.

Reply:
And with your fix nothing else uses the old format right so removing it is fine? 🙏

Reply:
we still support it on the feed side, but if you are not going to use it - it's ok.

Reply:
Removed :)